.gitignore

```diff
@@ -4,3 +4,5 @@ target/
 
 .gdb_history
 .gdbinit
+
+.idea
```

Comment:
This would do better in a system-specific gitignore. You can create one by setting the `core.excludesfile` setting to a gitignore file.

Reply:
Done :)